tutorials/standalone-tensorflow-raspberry-pi/index.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Setting Up a Standalone Tensorflow Prediction Device with Raspberry Pi and Google Cloud
```

Comment:
The is a capital 'F' in 'TensorFlow'.

I won't mark every instance in this review.